app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Foreign key already declared in the _User_ model, and hence no need to declare it again in a referenced model. Simply do something like:
```ruby
# app/models/comment.rb
belongs_to :author, class_name: 'User'

Reply:
OK, I will do well to make corrections immediately.

Reply:
- [ ] This comment has not been addressed yet